BrowserKit/Sources/Common/Constants/StandardImageIdentifiers.swift

```diff
@@ -21,7 +21,7 @@ public struct StandardImageIdentifiers {
         public static let bookmarkBadgeFillBlue50 = "bookmarkBadgeFillMediumBlue50"
         public static let cross = "crossMedium"
         public static let crossCircleFill = "crossCircleFillMedium"
-        public static let lock = "lockMedium"
+        public static let lockFill = "lockFillMedium"
```

Comment:
This image is not in the Acorn Github library - only images that are in Acorn should be added here. Additionally it seems the image is not used. So can you please remove the image and the reference to it here?